sw/device/sca/aes_serial/aes_serial.c

```diff
@@ -50,8 +50,13 @@ static const uint32_t kGpioCaptureTriggerLow = 0x00000;
 // RV Timer settings
 static const uint32_t kHart = (uint32_t)kTopEarlgreyPlicTargetIbex0;
 static const uint32_t kComparator = 0;
-static const uint64_t kTickFreqHz = 1000 * 1000;  // 1 MHz.
-static const uint64_t kDeadline = 500;            //  500us.
+static const uint64_t kDeadline = 200;  // 200 clock cycles at kClockFreqCpuHz
+// Caution: This number should be chosen to provide sufficient ample time.
+// Otherwise, Ibex might wake up while AES is still busy and disturb the
+// power measurement. Currently, we use a start trigger delay of 40 clock
```

Comment:
Just curious, is this delay implemented in HW? (adc offset in `util/device.py` is 0).

Reply:
Yes, it's the `SecAesAllowForcingMasks` parameter in `hw/top_earlgrey/rtl/top_earlgrey_cw305.sv`. That's different from the ADC offset in the scope. The former delays the start of the AES engine after receiving the software trigger. The latter delays the capture after receiving the hardware trigger.

Note, the hardware trigger signal is a combination of the GPIO (enabled below) AND and the AES busy signal. This way we have a super accurate trigger and don't need to realign capture traces.